pkg/restapi/issuer/operation/operations.go

```diff
@@ -332,6 +355,13 @@ func (o *Operation) createIssuerProfileHandler(rw http.ResponseWriter, req *http
 		return
 	}
 
+	_, err = o.createOIDCClient(profileData)
+	if err != nil {
+		commhttp.WriteErrorResponseWithLog(rw, http.StatusBadRequest,
```

Comment:
this does not seem to merit a BadRequest code. Looks more like InternalServerError.